static/app/views/dashboards/widgetCard/index.tsx

```diff
@@ -286,41 +287,31 @@ class WidgetCard extends Component<Props, State> {
       >
         {conditionalWrapWithDashboardsMEPProvider(
           <React.Fragment>
-            <WidgetCardPanel isDragging={false}>
-              <WidgetHeader>
-                <Tooltip
-                  title={widget.title}
-                  containerDisplayMode="grid"
-                  showOnlyOnOverflow
-                >
-                  <WidgetTitle>{widget.title}</WidgetTitle>
-                </Tooltip>
-                {this.renderContextMenu()}
-              </WidgetHeader>
-              {hasSessionDuration && SESSION_DURATION_ALERT}
-              {isWidgetInvalid ? (
-                <Fragment>
-                  {renderErrorMessage?.('Widget query condition is invalid.')}
-                  <StyledErrorPanel>
-                    <IconWarning color="gray500" size="lg" />
-                  </StyledErrorPanel>
-                </Fragment>
-              ) : noLazyLoad ? (
-                <WidgetCardChartContainer
-                  location={location}
-                  api={api}
-                  organization={organization}
-                  selection={selection}
-                  widget={widget}
-                  isMobile={isMobile}
-                  renderErrorMessage={renderErrorMessage}
-                  tableItemLimit={tableItemLimit}
-                  windowWidth={windowWidth}
-                  onDataFetched={this.setData}
-                  dashboardFilters={dashboardFilters}
-                />
-              ) : (
-                <LazyLoad once resize height={200}>
+            <VisuallyCompleteWithData
+              id="DashboardList-FirstWidgetCard"
+              hasData={(this.state.tableData?.length ?? 0) > 0}
```

Comment:
We should probably also check `this.state.seriesData` as well because the widget doesn't necessarily have to be a table